assets/semi-structured/clickhouse/config.yaml

```diff
@@ -0,0 +1,29 @@
+system_metric:
+  enable: true
+  memory:
+    ingest_polling_interval: 5
+    run_query_benchmark_polling_interval: 5
+
+container_id: clickhouse-clp-bench
+assets_path: /home/assets
+datasets_path: /home/datasets/mongod.log
+hot_run_warm_up_times: 3
+related_processes:
+  - clickhouse-server
+  - clickhouse-client
+  - clickhouse-watchd
+queries:
+  - '"JSON_EXISTS(raw, ''$.attr.tickets'')"'
```

Comment:
Is every query supposed to be wrapped in double-quotes? If so, that should be mentioned in the README, right? That said, do we need to wrap every query in double-quotes?